### PR TITLE
Re-design of the typing indicator

### PR DESCRIPTION
**Pull request by typfel**
## What changed
The typing indicator has moved from the left accessory view of the input bar to the top of input bar. The typing indicator has also been re-enabled for group conversations.

### CODE REVIEW

File: Wire-iOS Tests/TypingIndicatorViewTests.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
+//
+//  TypingIndicatorViewTests.swift
+//  Wire-iOS
+//
+//  Created by Jacob on 05/10/16.
+//  Copyright © 2016 Zeta Project Germany GmbH. All rights reserved.
```
Comment (daehn) [inline Wire-iOS Tests/TypingIndicatorViewTests.swift]: Missing copyright header

Comment (typfel) [inline Wire-iOS Tests/TypingIndicatorViewTests.swift]: 😨

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/TypingIndicatorView.swift [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        addSubview(expandingLine)
+        addSubview(container)
+        container.addSubview(nameLabel)
+        container.addSubview(animatedPen)
+        
+//        container.translatesAutoresizingMaskIntoConstraints = false
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/TypingIndicatorView.swift]: Can be removed

File: Wire-iOS.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -4721,10 +4709,12 @@
 				TargetAttributes = {
 					1EE85C4F1B5F96FB008E084B = {
 						CreatedOnToolsVersion = 6.4;
+						DevelopmentTeam = W5KEQBF9B5;
```
Comment (mikeger) [inline Wire-iOS.xcodeproj/project.pbxproj]: Should not be added I guess

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.m [hunk 4]
```diff
@@ -354,14 +355,19 @@ - (void)createEmojiButton
...
-    [self.typingView autoAlignAxisToSuperviewAxis:ALAxisVertical];
-    [self.typingView autoPinEdgeToSuperviewEdge:ALEdgeTop withInset:14];
+    self.typingIndicatorView = [[TypingIndicatorView alloc] init];
+    self.typingIndicatorView.translatesAutoresizingMaskIntoConstraints = NO;
+    self.typingIndicatorView.accessibilityIdentifier = @"typingIndicator";
+    self.typingIndicatorView.typingUsers = self.typingUsers.allObjects;
+    [self.typingIndicatorView setHidden:YES animated:NO];
+    
+    [self.inputBar  addSubview:self.typingIndicatorView];
+    [self.typingIndicatorView  autoConstrainAttribute:(ALAttribute)ALAxisHorizontal toAttribute:ALAttributeTop ofView:self.inputBar];
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.m]: Is the casting necessary?

Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/ConversationInputBarViewController.m]: I'm afraid so. PureLayout also does it internally.

**Pull request merged**